src/utils.py

```diff
@@ -29,6 +29,32 @@
 matplotlib.use("Agg")
 
 
+def get_aabb_volume(lo: Array, hi: Array) -> float:
+    """Simple utility function to compute the volume of an aabb.
+
+    lo refers to the minimum values of the bbox in the x,y and z axes,
+    while hi refers to the highest values
+    """
+    dimension = hi - lo
+    return dimension[0] * dimension[1] * dimension[2]
+
```

Comment:
add blank line (hmm, thought yapf handles this...)

Reply:
yeah; I thought so too. I guess not always?